### PR TITLE
Add Democracy

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: runtime/crab/polkadot-compatible-types.json [hunk 1]
```diff
@@ -243,6 +243,7 @@
 			"_enum": {
 				"Any": null,
 				"NonTransfer": null,
+				"ProxyType": null,
```
Comment (AsceticBear) [inline runtime/crab/polkadot-compatible-types.json:246]: A bit confused:

What are `polkadot-compatible-types.json` and `type.json` used for?  They have extractly the same content.

Comment (AurevoirXavier) [reply runtime/crab/polkadot-compatible-types.json:246]: No, their content are different

**Comment by AurevoirXavier** [comment]
Waiting https://github.com/darwinia-network/darwinia-common/pull/301

File: runtime/crab/polkadot-compatible-types.json (hunk 1 above)
Comment (AurevoirXavier) [reply runtime/crab/polkadot-compatible-types.json:246]: One for darwinia apps another for polkadot apps

**Review by HackFisher — CHANGES_REQUESTED** [review]
Need to bump Crab Runtime's Spec version

**Review by HackFisher — APPROVED** [review]

**Pull request merged**